src/main/download.py

```diff
@@ -1,4 +1,16 @@
 import os
+import logging
+
+logger = logging.getLogger()
+logger.setLevel(logging.DEBUG)
```

Comment:
Can we have the logger level configurable? This applies for both scripts

Reply:
It is done but just for model.py because they call a specific method in the download script